css/card.css

```diff
@@ -0,0 +1,205 @@
+    width: 80%;
+    max-width: 250px;
+    margin: var(--margin-width);
+    box-shadow: 0 0 10px 2px var(--shadow-color);
+    border-radius: 10px;
+    overflow: hidden;
+  }
+  .card:hover {
+    transition: 0.3s;
+    transform: scale(1.03);
+  }
+  
+  .card-badge {
+    padding: 0.1rem 0.4rem;
+    position: absolute;
+    color: var(--danger);
+    cursor: pointer;
+  }
+  .white-bg {
+    background-color: var(--white);
+  }
+  .pink-bg {
+    background-color: pink;
+  }
+  .badge-btn {
+    position: absolute;
+    color: var(--white);
+    cursor: pointer;
+  }
+  .left {
+    left: 0;
+    top: 1rem;
+  }
+  .right {
+    right: 0;
+    top: 1rem;
+  }
+  .top-right {
+    right: 1rem;
+    top: 1rem;
+  }
+  span {
+    margin: 1rem 0rem;
+  }
+  .card-with-text {
+    display: flex;
+    flex-direction: column;
+    font-size: 1rem;
+  }
+  .card-text-title {
+    padding-bottom: 0.3rem;
+  }
+  .desc {
+    padding-bottom: 0.3rem;
+  }
+  .bold {
+    font-weight: bold;
+  }
+  .card-horizontal {
+    display: flex;
+    justify-content: space-evenly;
+    align-items: center;
+    width: 80%;
+    max-width: 400px;
+  }
+  .img-horizontal {
+    height: 80%;
+    width: 40%;
+    min-width: 100px;
+    max-height: 100px;
+    border-radius: 10px 10px 0 0;
+  }
+  .text-overlay-card-image {
+    height: 100%;
+    width: 100%;
+    cursor: pointer;
+  }
+  .card-with-text-overlay {
+    display: flex;
+    height: 100%;
+    width: 70%;
+    font-size: 1rem;
+  }
+  .overlay-text {
+    position: absolute;
+    font-weight: 600;
+    font-size: large;
+    color: black;
+    left: 50%;
+    top: 35%;
+    transform: translate(-50%, -50%);
+    background-color: white;
+    padding: 0.5rem 2rem;
+  }
+  .text-card {
+    border: 1px solid var(--gray-border);
+    text-align: justify;
+    width: 50%;
+  }
+  .card-details {
+    margin: 0;
+    padding: 0.5rem 0.5rem;
+  }
+  .card-details > * {
+    margin: 0;
+  }
+  .card-btn {
+    display: flex;
+    justify-content: center;
+    padding-top: 0.2rem;
+  }
+  .card-btn > * {
+    margin: 0.5rem;
+  }
+  .grayed-img {
+    opacity: 0.5;
+  }
+  
+  .btn-primary {
+    border: 1px solid var(--gray-border);
+    padding: 0.5rem 1rem;
+    color: var(--white);
+    border-radius: 7px;
+    cursor: pointer;
+  }
+  .btn-text {
+    background-color: transparent;
+    cursor: pointer;
+    border: 0;
+    line-height: 1.75;
+    font-weight: 600;
+    letter-spacing: 0.02857em;
+  }
+  .btn-secondary {
+    background-color: transparent;
+    cursor: pointer;
+    border: 0;
+    border: 1px solid;
+    border-radius: 7px;
+    padding: 0.45rem 1rem;
+  }
+  .btn-bg-color {
+    background-color: var(--primary-color);
+  }
+  @media only screen and (min-width: 1100px) {
+    .text-card {
+      width: 60%;
+    }
+  }
+  /* @media only screen and (min-width: 601px) and (max-width: 800px) {
+    .card {
+      width: 40%;
+    }
+  
+    .text-card {
+      width: 60%;
```

Comment:
Suggestion to delete comments if not required

Reply:
I've made the required changes as per suggestions.